src/RoutingControllersOptions.ts

```diff
@@ -85,4 +86,28 @@ export interface RoutingControllersOptions {
      */
     currentUserChecker?: CurrentUserChecker;
     
+    /**
+     * Default settings
+     */
+    defaults?: {
+        /**
+         * If set, all null responses will return specified status code by default
+         */
+        nullResultCode?: number;
+
+        /**
+         * If set, all undefined responses will return specified status code by default
+         */
+        undefinedResultCode?: number;
+
+        /**
+         * Default param options
+         */
+        paramOptions?: {
```

Comment:
I agree with @19majkel94 it's more future proof to add it as an object, it would save as a few release where we would have to keep the deprecated version working as well. 

Btw is this PR ok except this part?